skymp5-client/src/front/skympClient.ts

```diff
@@ -447,21 +446,35 @@ export class SkympClient {
     this.sendTarget.send({ t: MsgType.Host, remoteId }, false);
   }
 
-  private sendInputs() {
+  private updateData()
+  {
     const hosted =
       typeof storage["hosted"] === typeof [] ? storage["hosted"] : [];
     const targets = [undefined].concat(hosted as any);
-    //printConsole({ targets });
+    // removing all stop hosted objects from usedRefID array
+    deathSystem.usedRefID.filter((value) => targets.includes(value))
+
     targets.forEach((target) => {
-      this.sendMovement(target);
-      this.sendAnimation(target);
-      this.sendLook(target);
-      this.sendEquipment(target);
-      this.sendActorValuePercentage(target);
+      this.prepareSendInputs(target);
+      this.tryMakeActorImmortal(target);
     });
+
     this.sendHostAttempts();
   }
 
+  private prepareSendInputs(target : undefined) {
+    this.sendMovement(target);
+    this.sendAnimation(target);
+    this.sendLook(target);
+    this.sendEquipment(target);
+    this.sendActorValuePercentage(target);
+  }
+
+  private tryMakeActorImmortal(target : undefined)
```

Comment:
Making actors immortal is not related to the inputs we send. Please write the code that is responsible for **correctly show other characters** in `view.ts`